src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -96,16 +105,22 @@ class ReportActionItemMessageEdit extends React.Component {
     /**
      * Save the draft of the comment. This debounced so that we're not ceaselessly saving your edit. Saving the draft
      * allows one to navigate somewhere else and come back to the comment and still have it in edit mode.
+     * @param {String} newDraft
      */
-    debouncedSaveDraft() {
-        saveReportActionDraft(this.props.reportID, this.props.action.reportActionID, this.state.draft);
+    debouncedSaveDraft(newDraft) {
+        saveReportActionDraft(this.props.reportID, this.props.action.reportActionID, newDraft);
     }
 
     /**
      * Save the draft of the comment to be the new comment message. This will take the comment out of "edit mode" with
      * the new content.
      */
     publishDraft() {
+        /**
+         * To prevent re-mount after user saves edit before debounce duration, we cancel debounce here.
```

Comment:
```suggestion
         * To prevent re-mount after user saves edit before debounce duration (example: within 1 second), we cancel debounce here.
```